R/ratio_calculations.R

```diff
@@ -215,6 +236,12 @@ calculate_ratios.data.frame <- function(
         ifelse(n > 1, paste0("RA", test_parameter, " (mean)"), paste0("RA", test_parameter))
       }
     ) %>%
+    # Make sure all basic columns are still character (even when empty)
```

Comment:
nitpick: consider adding the issue reference so we know why this "odd" part of the code was done
```suggestion
    # Make sure all basic columns are still character (even when empty #730)
```